services/galley/src/Galley/API.hs

```diff
@@ -904,23 +906,49 @@ sitemap = do
         .&. capture "uid"
         .&. accept "application" "json"
 
-    get "/i/teams/:tid/legalhold" (continue LegalHold.getEnabled) $
+    get "/i/users/:uid/team/members" (continue getBindingTeamMembers) $
+        capture "uid"
+
+    get "/i/users/:uid/team" (continue getBindingTeamId) $
+        capture "uid"
+
+    -- Start of team features; enabling this should only be
+    -- possible internally. Viewing the status should be allowed
+    -- for any admin
+
+    get "/teams/:tid/features/legalhold" (continue Teams.getLegalholdStatus) $
+        zauthUserId
+        .&. capture "tid"
+        .&. accept "application" "json"
+
+    get "/teams/:tid/features/sso" (continue Teams.getSSOStatus) $
+        zauthUserId
+        .&. capture "tid"
+        .&. accept "application" "json"
+
+    get "/i/teams/:tid/features/legalhold" (continue Teams.getLegalholdStatusInternal) $
         capture "tid"
         .&. accept "application" "json"
 
-    put "/i/teams/:tid/legalhold" (continue LegalHold.setEnabled) $
+    put "/i/teams/:tid/features/legalhold" (continue Teams.setLegalholdStatusInternal) $
         capture "tid"
         .&. jsonRequest @LegalHoldTeamConfig
         .&. accept "application" "json"
 
-    get "/i/users/:uid/team/members" (continue getBindingTeamMembers) $
-        capture "uid"
+    get "/i/teams/:tid/features/sso" (continue Teams.getSSOStatusInternal) $
+        capture "tid"
+        .&. accept "application" "json"
 
-    get "/i/users/:uid/team" (continue getBindingTeamId) $
-        capture "uid"
+    put "/i/teams/:tid/features/sso" (continue Teams.setSSOStatusInternal) $
+        capture "tid"
+        .&. jsonRequest @SSOTeamConfig
+        .&. accept "application" "json"
+
+    -- End of team features
 
     get "/i/test/clients" (continue getClients)
         zauthUserId
+    -- TODO: What is this endpoint? Is this used anywhere?
```

Comment:
Grep or ag helps finding usage, it's used here: https://github.com/wireapp/wire-server/blob/develop/services/brig/test/integration/API/User/Client.hs#L319
Maybe you could add a comment and remove the TODO.

Reply:
:+1:

(will do that in #817)